codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/config/ClientConfig.java

```diff
@@ -258,7 +234,7 @@ public Builder defaultPrimitiveValue(String value) {
         }
 
         /**
-         * @param rubyDefaultBlock a single, dynamic default value to use.
+         * @param rubyDefaultBlock a single, dynamic default value to use. Assumes a proc with a cfg parameter.
```

Comment:
This doc is a little confusing.  Instead of "Assumes" be more specific about exactly what you must provide (a callable that accepts a single value).

Reply:
It's not a callable. It's the proc body